src/components/BookComponent/eachBook.module.css

```diff
@@ -0,0 +1,77 @@
+.leftBody {
+  border: 2px solid rgb(226, 226, 226);
+  height: 70%;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.ulBody {
+  font-family: Montserrat, 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  width: 10rem;
+  height: 3.625rem;
+  margin: 30px 20px;
+}
+
+.ulBody ul {
+  list-style: none;
+}
+
+.category {
+  opacity: 0.5;
+  font-family: Montserrat, 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.title {
+  width: 12.313rem;
```

Comment:
- [ ] Please kindly ensure that your design meets the project design requirements(colors, font, padding, button, spacing etc)

### Your Design
![image](https://user-images.githubusercontent.com/43995830/179132267-addc58f6-8521-4618-bd4f-030a89ca42d3.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/179132219-03368dc8-06f0-46f7-a6ac-3b320eae7cbf.png)

Reply:
@AnselemOdims Thank you, I have fixed this up.